main/ertp/ERTP-Guide/AmountMath.md

```diff
@@ -0,0 +1,193 @@
+# Amount Math
+
+![AmountMath methods](AmountMath.svg) 
+
+`issuer` objects must be able to add and withdraw assets from a
+`purse` and manipulate `payment` amounts. This 
+requires being able to add and subtract digital assets. To do this,
+and other operations on `amount` objects, an `issuer` uses a set 
+of `amountMath` methods. 
+
+In addition to math operations, `amountMath` methods check on their
+arguments' brands, throwing an error if the wrong `brand` was used.
+
+There are three different kinds of `amountMath`s, each of which
+implements the same set of methods. When you create an `issuer`, you
+do so for a specific `brand`. You thus have to specify which kind of
+`amountMath` that `brand` uses. The 
+correct kind is automatically used whenever an `amountMath` method
+is used on `amount` objects with that `brand`. The kinds are: 
+- `nat`: Used with fungible assests (i.e. natural numbers).
+- `strSet`: Used with non-fungible assets (i.e. strings).
+- `set`: Used with sets of objects, primarily non-fungible assets.
+
+`produceIssuer(allegedName, amountMathKind)` creates a new `issuer`.
+The second, optional, argument specifies which kind
+of `amountMath` your contract uses for the `brand` in a one-to-one
+association with the new `issuer`. It defaults to `nat` if not given. For example: 
+```js
+produceIssuer('quatloos`); // Defaults to 'nat'
+produceIssuer('quatloos', 'strSet');
+produceIssuer('quatloos, 'set');
+```
+`AmountMath` has:
+- Three information getting methods (get brand, get value, and get
+`amountMath` kind)
+- Four comparison methods (is an amount empty, is an amount valid,
+are two amounts equal, and is one amount greater than or equal to another)
+- Two manipulator methods (add and subtract amounts)
+- One amount creation method (create an `amount` with the
+  `amountMath`'s associated `brand`)
+
+The following is a brief description and example of each `amountMath` method. For
+more detail, click the method's name to go to its entry in the ERTP
+API Reference.
+
+- **Information Getting Methods**
+  - [`amountMath.getBrand()`](https://agoric.com/documentation/ertp/api/amount-math.html#amountmath-getbrand)
+    - For this `amountMath`, return its `brand`.
+    - ```js
```

Comment:
The leading dash makes this render as a second bullet under the previous.  
The fact that the triple-backtick is not at the start of the line breaks the code formatting.
(here, and pervasively below.)